week-4/Homework/mandatory/4-groceries.js

```diff
@@ -40,8 +41,12 @@ Exercise 2:
 // Gather weekend item names into this array
 let weekendGroceriesToBuy = [];
 
-
-
+for (let meal in weeklyMealPlan) {
+  if (meal === "saturday" || meal === "sunday") {
+    weekendGroceriesToBuy.push(weeklyMealPlan[meal]);
```

Comment:
this will cause the same issue (array of arrays)
try to think about using concatination (array concat)